internal/convert.go

```diff
@@ -85,13 +85,16 @@ const (
 	Widened
 	Time
 	StringOverflow
+	NameConvention
```

Comment:
Nit: should we try for a more informative name for this issue? How about IllegalName?

Reply:
will do.